### PR TITLE
Frontend boilerplate

### PR DESCRIPTION
**Pull request by trumko**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+    </head>
+    <body>
+      <div id="root">Loading...</div>
+        <script type="text/javascript" src="bundle.js" charset="utf-8"></script>
```
Comment (ktv18) [inline public/index.html]: remove charset attr
script tag should have the same indent as div does

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+class App extends React.Component {
+  constructor(props) {
```
Comment (ktv18) [inline src/components/App.js]: you don't need this constructor since it just call parent's constructor

File: src/components/App.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    super(props);
+  }
+
+  render() {
+    return (
+      <div >
```
Comment (ktv18) [inline src/components/App.js]: extra space in div tag

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App';
+import style from './styles/main.scss'
+
```
Comment (ktv18) [inline src/index.js:5]: remove extra empty line

File: src/styles/main.scss [hunk 5]
```diff
@@ -0,0 +1,7 @@
+@import 'flexboxgrid';
+
+#root {
+    h1 {
```
Comment (ktv18) [inline src/styles/main.scss]: `h1 { color: red };`

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+var path = require('path')
```
Comment (ktv18) [inline webpack.config.js]: missing semicolon
use const instead of var

File: public/index.html (hunk 1 above)
Comment (trumko) [reply public/index.html]: I've also deleted `type="text/javascript"` as we don't need it in HTML5

**Review by ktv18 — APPROVED** [review]

**Review by ktv18 — APPROVED** [review]

**Pull request merged**